src/expr/src/relation.rs

```diff
@@ -2754,33 +2754,96 @@ impl AggregateExpr {
                     .call_unary(UnaryFunc::RecordGet(scalar_func::RecordGet(0)));
 
                 // Extract the input value
-                let expr = tuple.call_unary(UnaryFunc::RecordGet(scalar_func::RecordGet(1)));
-
-                // If the window frame includes the current (single) row, evaluate the aggregate on
-                // that row. Otherwise, return the default value for the aggregate.
-                let value = if window_frame.includes_current_row() {
-                    AggregateExpr {
-                        func: (**wrapped_aggregate).clone(),
-                        expr,
-                        distinct: false, // We have just one input element; DISTINCT doesn't matter.
-                    }
-                    .on_unique(input_type)
```

Comment:
This evaluates the aggregate that is wrapped inside the window function. E.g., for `sum(...) OVER (...)`, this would evaluate the `sum` on that one row that is in the window partition.